docs/5.x/cluster.md

```diff
@@ -1656,7 +1656,7 @@ With this configuration, the Ops Center cluster will provide a single Kubernetes
 service called `gravity-public` configured to serve both user and cluster
 traffic:
 
-```bash
+```bsh
```

Comment:
Are you sure `bsh` is the correct snippet type? Not `bash`? Cause all these snippets do not display correctly (I'm looking at the staging):

![screenshot from 2018-10-30 15-01-19](https://user-images.githubusercontent.com/339322/47753420-b06b8480-dc54-11e8-999b-f0bcfcfd98b5.png)

Note "$" sign is missing. And here "#" is treated like a comment:

![screenshot from 2018-10-30 15-01-56](https://user-images.githubusercontent.com/339322/47753445-c8430880-dc54-11e8-94ef-904c9e4b91d1.png)

Reply:
Yes, 100% sure. It's a different code formatter now and the it looks for "lang-bsh" instead of the old "bash".

Reply:
Okay. Well, somehow snippets are still not rendered correctly... (see my comment above about missing $ character)